### ISSUE TITLE
Thruster Shrouds

### ISSUE THREAD

**Issue by amcaleer271** [uf-mil-mechanical/tasks#17]
The thrusters on NaviGator must be shrouded to ensure safety during the competition (Handbook, pg. 34, item 5). 

These shrouds should prevent large items from entering the thrusters, but they do not need to completely cover them.

**Comment by adamhamdan8** [comment]
Previous years they used something silly like a wire basket, so the requirement is not super strict, but preferably something nice

**Comment by csachs32** [comment]
I can work on this

**Comment by csachs32** [comment]
Designed two parts that connect together via 1/4-20 bolts that shroud the thruster. The two parts and an assembly with a thruster can be located in the "Thruster Shroud" Folder in "Navigator AMS". A picture of the design is attached.
![Image](https://github.com/user-attachments/assets/f411de9a-1e9b-4f45-b1bb-4d9c8c3fa34b)

**Comment by adamhamdan8** [comment]
Taking a quick look now, that looks good! I can't really think of a more elegant solution for right now, and this would be quick to do. We can print a prototype and see what we like and what we don't like!

**Comment by csachs32** [comment]
Uploaded a prototype to the 3D printer and am waiting for it to print/ next time I go to the lab to test it out.

**Comment by csachs32** [comment]
Printed the first half of the shroud and will start the second half print today.

**Comment by csachs32** [comment]
Printed the first prototype of the shrouds, but they do not fit around the thrusters correctly. Need to make changes to the design.

**Comment by csachs32** [comment]
Made changes to the design, but now the shrouds cannot print on the 3D printer because they are too big. Currently troubleshooting this.![Image](https://github.com/user-attachments/assets/018f6aa6-2242-47b6-a7f1-0f072068870f)

**Comment by csachs32** [comment]
We might have to print these at the rapid proto lab. Waiting on the leads to discuss this dilemma.

**Comment by csachs32** [comment]
Split each side of the thruster in half so that there are now 4 parts for each shroud. Sent the parts to the print queue, and once printed, will weld each part together to test the prototype.

**Comment by adamhamdan8** [comment]
Lots of things are in the print queue, so this will be on hold for a bit!

**Comment by csachs32** [comment]
Printed final shrouds for each thruster (and an extra shroud), and welded one thruster together to mount to NaviGator.

**Comment by adamhamdan8** [comment]
We had to macgyver like four different solutions. The current one is still on navigator for all four thrusters. Created by @adamhamdan8 @amcaleer271 @space-cat-explorer Should be sufficient for the future
